### PR TITLE
serpl: new port

### PR DESCRIPTION
**Pull request by yassinebridi**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 14.6
Xcode 15.3.0

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: devel/serpl/Portfile [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+PortSystem          1.0
+PortGroup           cargo   1.0
+PortGroup           github  1.0
+
+github.setup        yassinebridi serpl 0.3.3
+github.tarball_from archive
+revision            0
+
+homepage            https://github.com/yassinebridi/serpl
```
Comment (herbygillot) [inline devel/serpl/Portfile:11]: `homepage` isn't required here as it is implicitly set by `github.setup` above

**Comment by herbygillot** [comment]
Hello @yassinebridi, thank you for this submission.  Just one minor nitpick, but this looks good otherwise.

**Comment by yassinebridi** [comment]
Thanks @herbygillot, fixed and squashed the commit.

**Comment by herbygillot** [comment]
Looks good and merged. Thank you for your contribution to MacPorts, @yassinebridi.

**Pull request merged**